.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+-   repo: git@github.com:Yelp/detect-secrets
+    rev: v0.13.1
```

Comment:
Since this is a new repo, I think it would be nice to bump up to `detect-secrets V1.0.3`. This will make it easier to scan for particular plugins or list them all (versus a really long command) because they added these nifty flags:  --list-all-plugins  and --disable-plugin. I upgraded Atlas and WebAPI to this as well.

Reply:
i'm not sure what's going on but i'm running into python errors when updating the detect-secrets version and running `pre-commit run -a` locally 🤔

Reply:
There is an open issue, `detect-secret-hooks failing with latest version upgrade`, over at the detect-secrets repo, a possible versions issue?